style.css

```diff
@@ -0,0 +1,316 @@
+*{
+box-sizing:boder-box;
+margin:0;
+padding:0;
+/* max-width:1023px; */
+}html{
+font-family: "Helvetica Neue", Arial, Helvetica, sans-serif;
+}
+img{
+max-width:100%;
+}
+body{
+background-color:#E5EAE9;
+}
+.header{
+padding:.5rem 0 .5rem 2.25rem;
+/* margin-left:5%; */
+background-color: #41EAD4;
+}
+.nav__bar {
+    display: flex;
+    flex-direction:row;
+    flex-wrap:wrap;
+    justify-content:space-evenly;
+    align-items:flex-start;
+    align-content:center;
+    background-color:#495867;
+}
+.nav__bar__item{
+    list-style-type:none;
+    padding: .5rem 1.25rem;
+    color: #D0FEF5;
+}
+.nav__bar a{
+    text-decoration: none;   
+}
+.nav__bar a:hover{
+    text-decoration: underline;
+}
+.picture__profile__container{
+    display: flex;
+    flex-direction:row;
+    flex-wrap:wrap;
+    justify-content:center;
+    align-items:flex-start;
+    align-content:center;
+}
+.image{
+    display: block;
+    padding:20px 0px 5px 0px;
+    width:75%; 
+    margin-left: auto;
+    margin-right: auto;
+}
+.profile__picture{
+    width:75%;
+    border-radius:50%
+}
+.about__me__all{
+    margin-left:5%;
+    margin-bottom:20px;
+}
+.social__media{
+    display: flex;
+    flex-direction:row;
+    flex-wrap:wrap;
+    justify-content:space-evenly;
+    align-items: center;
+    align-content:center;
+    /* padding: .5rem 0; */
+    background-color:#495867;
+}
+.wrapper{
+    min-height: calc(100vh - 100px);
```

Comment:
This is some advanced CSS! Good work here. I'd love to hear the story behind how you figured this out.